code/stable/dblpendulum/SRS/JSON/DblPendulum_SRS.ipynb

```diff
@@ -155,7 +155,7 @@
     "## Purpose of Document\n",
     "<a id=\"Sec:DocPurpose\"></a>\n",
     "\n",
-    "The primary purpose of this document is to record the requirements of the Pendulum. Goals, assumptions, theoretical models, definitions, and other model derivation information are specified, allowing the reader to fully understand and verify the purpose and scientific basis of DblPendulum. With the exception of [system constraints](#Sec:SysConstraints), this SRS will remain abstract, describing what problem is being solved, but not how to solve it.\n",
+    "The primary purpose of this document is to record the requirements of DblPendulum. Goals, assumptions, theoretical models, definitions, and other model derivation information are specified, allowing the reader to fully understand and verify the purpose and scientific basis of DblPendulum. With the exception of [system constraints](#Sec:SysConstraints), this SRS will remain abstract, describing what problem is being solved, but not how to solve it.\n",
```

Comment:
"DblPendulum" (and "SglPendulum") should be added to the table of abbreviations.